meinberlin/apps/activities/migrations/0004_alter_activity_description.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.20 on 2023-10-12 14:46
+
+from django.db import migrations
+import django_ckeditor_5.fields
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("meinberlin_activities", "0003_add_helptext"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="activity",
+            name="description",
+            field=django_ckeditor_5.fields.CKEditor5Field(verbose_name="Description"),
+        ),
+    ]
```

Comment:
maybe first add a new field, e.g `description_ck5`, then do the data migration from (old) `description` to new one, then drop `description` field, and in a new migration file, rename `description_ck5` to `description`

Reply:
This migration works fine already I think. I tagged you on the a4 pr as I don't really know what to do there.